webpush/utils.py

```diff
@@ -5,28 +5,28 @@
 from pywebpush import WebPushException, webpush
 
 
-def send_notification_to_user(user, payload, ttl=0):
+def send_notification_to_user(user, payload, ttl=0, headers={}):
     # Get all the push_info of the user
 
     push_infos = user.webpush_info.select_related("subscription")
     for push_info in push_infos:
-        _send_notification(push_info.subscription, payload, ttl)
+        _send_notification(push_info.subscription, payload, ttl, headers=headers)
 
 
-def send_notification_to_group(group_name, payload, ttl=0):
+def send_notification_to_group(group_name, payload, ttl=0, headers={}):
```

Comment:
It is not good idea to take [default mutable value](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/). Can you please change it?